scripts/crontab/crontab.tpl

```diff
@@ -40,20 +40,19 @@ HOME=/tmp
 0 8 * * * %(django)s update_product_details
 
 # Once per day after metrics import is done
-00 17 * * * %(z_cron)s update_addon_download_totals
-05 17 * * * %(z_cron)s weekly_downloads
-55 17 * * * %(z_cron)s update_global_totals
-00 18 * * * %(z_cron)s update_addon_average_daily_users
-30 18 * * * %(z_cron)s index_latest_stats
-45 18 * * * %(z_cron)s update_addons_collections_downloads
-
-# Update ADI metrics from HIVE.
-# Once per day after 1000 UTC (after hive queries + transfer is done)
-NETAPP_STORAGE_ROOT='/var/tmp'
-00 16 * * * %(django)s update_counts_from_file
-30 16 * * * %(django)s download_counts_from_file
-00 17 * * * %(django)s theme_update_counts_from_file
-00 18 * * * %(django)s update_theme_popularity_movers
+00 9 * * * %(z_cron)s update_addon_download_totals
+05 9 * * * %(z_cron)s weekly_downloads
+55 9 * * * %(z_cron)s update_global_totals
+00 10 * * * %(z_cron)s update_addon_average_daily_users
+30 10 * * * %(z_cron)s index_latest_stats
+45 10 * * * %(z_cron)s update_addons_collections_downloads
+
+# Update ADI metrics from S3.
+# Once per day after 0800 UTC
```

Comment:
Why the change in time? Is there a guarantee that the files will be generated and copied over to either S3 (or EFS if we need to not use s3) by this time?

Reply:
Due to performance and capacity issues on the IT Hadoop cluster the MapReduce job was taking a very long time to complete. We have pushed the stats import later and later in the day to account for this.

I've made an optimization to the MapReduce job and ever since the job has been running in AWS EMR it has completed before 0800 UTC [1]. I won't call this a guarantee but it's been consistent. 

We can run it later in the day but I think most users would like to see stats sooner than later. 

[1] https://workflow.telemetry.mozilla.org/admin/airflow/tree?dag_id=mango_log_processing